app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,15 @@
+class PostsController < ApplicationController
+  private
+  def create
+    @Posts = Post.create!(post: params[:post])
+    # grab the user input for params
+    # create a new post from that input
+    # redirect back to the index
+    redirect_to :back
```

Comment:
If you're always going to redirect to the index, I would make that explicit here (instead of `:back`).